default.yaml

```diff
@@ -9,6 +9,10 @@ displayCurrency:
 # username of the account that will be used to fund rewards
 funder: "FunderWallet"
 
+dealer:
+  usd:
+    hedgingEnabled: false
```

Comment:
more than `hedgingEnabled`, should it be a broader setting `usdAvailable`?

Reply:
(note: I know I was the one using usd specifically, but as we start to see project that could use eur as a store of value, I'm wondering if we should rename usd to fiat in a follow up PR. I initially did the distinction of using usd to separate with the displayCurrency, but that was probably a bit short sighted.)

Reply:
I agree this is a valid inquiry - but this PR is already huge... lets try to push as many corrections as possible to follow up PRs and focus here on critical things.

Comments are still useful as reminders of course.